lib/crowbar/init/application.rb

```diff
@@ -219,6 +223,19 @@ def wait_for_crowbar
         end
       end
 
+      def migrate_database
```

Comment:
maybe we can be more specific which file is missing, and fail earlier like this:

```ruby
["data.yml", "schema.rb"].each do |dbfile|
  next if File.exist?("#{crowbar_framework_path}/db/#{dbfile}")
  logger.debug("Could not find #{crowbar_framework_path}/db/#{dbfile}")
  return false
end
run_cmd("cd /opt/dell/crowbar_framework && RAILS_ENV=production bin/rake db:load")
```